runtime_test.go

```diff
@@ -2971,6 +2971,76 @@ func TestExceptionWithinAppliedObjectFunc(t *testing.T) {
 	}
 }
 
+func TestExceptionStringifyError(t *testing.T) {
+	runtime := New()
+
+	for _, tc := range []struct {
+		description    string
+		exceptionVal   func() Value
+		expectedString string
+	}{
+		{
+			description: "Bool",
+			exceptionVal: func() Value {
+				return valueBool(true)
+			},
+			expectedString: "true",
+		},
+		{
+			description: "Int",
+			exceptionVal: func() Value {
+				return valueInt(1234)
+			},
+			expectedString: "1234",
+		},
+		{
+			description: "String",
+			exceptionVal: func() Value {
+				return newStringValue("hello")
+			},
+			expectedString: "\"hello\"",
+		},
+		{
+			description: "Null",
+			exceptionVal: func() Value {
+				return valueNull{}
+			},
+			expectedString: "null",
+		},
+		{
+			description: "Undefined",
+			exceptionVal: func() Value {
+				return valueUndefined{}
+			},
+			expectedString: "undefined",
+		},
+		{
+			description: "Object",
+			exceptionVal: func() Value {
+				testObject := runtime.NewObject()
+				testObject.Set("foo", "bar")
+				testObject.Set("fizz", "buzz")
+
+				return testObject
+			},
+			expectedString: "{\"foo\":\"bar\",\"fizz\":\"buzz\"}",
```

Comment:
[q] are goja objects always ordered? just want to make sure the keys in this string will always be in the same order.

Reply:
I am looking to see if I can figure this out definitively. Anecdotally, I ran the tests with `-count=30` and did not see any changes in the order.

Reply:
good enough for me

Reply:
It uses a go map under the hood, whose iteration order is not guaranteed. I think I will try to update this. Probably just going to break it into a separate test and get a little clever with the string comparison.

Reply:
Got a new test up there, let me know what you think